hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/TokensConfig.java

```diff
@@ -60,10 +60,10 @@ public record TokensConfig(
         @ConfigProperty(value = "maxMetadataBytes", defaultValue = "100") @NetworkProperty int tokensMaxMetadataBytes,
         @ConfigProperty(value = "balancesInQueries.enabled", defaultValue = "true") @NetworkProperty
                 boolean balancesInQueriesEnabled,
-        @ConfigProperty(value = "airdrops.enabled", defaultValue = "false") @NetworkProperty boolean airdropsEnabled,
-        @ConfigProperty(value = "airdrops.cancel.enabled", defaultValue = "false") @NetworkProperty
+        @ConfigProperty(value = "airdrops.enabled", defaultValue = "true") @NetworkProperty boolean airdropsEnabled,
+        @ConfigProperty(value = "airdrops.cancel.enabled", defaultValue = "true") @NetworkProperty
                 boolean cancelTokenAirdropEnabled,
-        @ConfigProperty(value = "airdrops.claim.enabled", defaultValue = "false") @NetworkProperty
+        @ConfigProperty(value = "airdrops.claim.enabled", defaultValue = "true") @NetworkProperty
```

Comment:
Are these config changes to be merged to develop in this PR?

Reply:
Those should have already been enabled by default on develop @david-bakin-sl . This feature is going in 0.54. I have changed the title of the PR 🙂

Reply:
Changing the title makes this make sense, TY!

Reply:
Thank you!